command/cp.go

```diff
@@ -270,7 +267,7 @@ func (c Copy) Run(ctx context.Context) error {
 	waiter.Wait()
 	<-errDoneCh
 
-	return merror
+	return nil
```

Comment:
If you don't return an error here, exit code of all commands will be 0 (success). Old versions of s5cmd(<v1.0.0) were working like this and we changed this behavior after version 1.0.0. Please find a better way.